src/App.tsx

```diff
@@ -1,10 +1,14 @@
+import 'react-native-gesture-handler';
 import React from 'react';
-import {View, Text} from 'react-native';
-import Home from './screens/Home';
+import {NavigationContainer} from '@react-navigation/native';
+import Routes from './navigation';
 
-// import { Container } from './styles';
+function App() {
+    return (
+        <NavigationContainer>
+            <Routes />
+        </NavigationContainer>
+    );
+}
```

Comment:
Switch it to an arrow function and dont forget to export by default

Reply:
resolved in commit a71461893b6fea906309d9e3507e0c49e0d139f5